### DISCUSSION TITLE
Add button to take me back to the actions PR

### DISCUSSION THREAD

**Discussion by basickarl** [community/community#35072] [Actions]
<img width="1784" alt="image" src="https://user-images.githubusercontent.com/10507098/193811775-1600c9fe-2619-49be-9d21-8c7188344deb.png">
 I spend a lot of my time fixing GitHub Action CI's and every-time I am in a "run" I would like to be able to return to the PR (if it is connected to a PR) or to the branch from which I clicked on to get to that run in the first place. a simple button in the top right would be great and would save me a lot of time.

**Comment by classabbyamp** [comment]
I also have this experience. I spend a lot of time going PR -> CI log -> PR and I am forced to use the browser back button because it takes me from the Checks tab in the PR to the Actions tab of the repo if I click on a different job than the one that was first loaded

Example:

first page I see, if I click on the Checks tab:
![image](https://user-images.githubusercontent.com/5366828/195654734-046c4c9b-1f49-4fd5-ad56-67ae8728f9d8.png)

Page I get redirected to if I click on one of the listed jobs:
![image](https://user-images.githubusercontent.com/5366828/195654784-185ee338-7def-4ce8-9992-133950dccd32.png)
